### PR TITLE
feat(EditorialQuote): Port EditorialQuote from monolith

### PR DESCRIPTION
**Pull request by rogervera**
# To be merged after `rv-editorial-image`

## Create EditorialQuote used for Editorial layouts

**Editorial Quote:**
A pull quote for use in an editorial page.

**Editorial Molecules?**
These molecules are used to build editorial layouts, such as [Articles](https://www.codecademy.com/resources/editorials/test-article), [Listicles](https://www.codecademy.com/resources/editorials/test-listicle), or [Learner Story](https://www.codecademy.com/learner-stories/data-science/1ogiwGOCDJNRTHczbLsyn8) pages.

### Current Monolith Use example
Screen Shot 2020-04-29 at 10 11 17 AM

### Storybook example
Screen Shot 2020-04-29 at 10 11 36 AM

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.12-alpha.533e84.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea98cff507922bd889f723d--codecademy-styleguide.netlify.app>https://5ea98cff507922bd889f723d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea98cff507922bd889f723d>Deploy Logs</a></p>

File: packages/brand-components/src/EditorialMolecules/EditorialQuote/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@import "~@codecademy/gamut-styles/utils";
+
+.quoteContainer {
+  padding-top: 2rem;
```
Comment (codecaaron) [inline packages/brand-components/src/EditorialMolecules/EditorialQuote/styles.module.scss:4]: For simplicities sake I think we can move this padding to original element.  And adjust the positioning accordingly.

That way we can colocate changes that are media dependent.

File: packages/brand-components/src/EditorialMolecules/EditorialQuote/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import s from './styles.module.scss';
+
+export type EditorialQuoteProps = {
+  quote: string;
+};
+
+export const EditorialQuote: React.FC<EditorialQuoteProps> = ({ quote }) => (
+  <div className={s.quoteContainer}>
+    <div className={s.quote}>{quote}</div>
+  </div>
```
[the remark this answers is not in the document]
Comment (rogervera) [reply packages/brand-components/src/EditorialMolecules/EditorialQuote/index.tsx:11]: And remove the quotes? `q:before::, q::after{content:''}`
Not opposed. Can make the change.

File: packages/brand-components/src/EditorialMolecules/EditorialQuote/styles.module.scss (hunk 1 above)
Comment (rogervera) [reply packages/brand-components/src/EditorialMolecules/EditorialQuote/styles.module.scss]: Good point. Changed in latest commit, and updated the story to reflect this.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.4.0-alpha.72c46b.0<br/></pre>

File: packages/brand-components/src/EditorialMolecules/EditorialQuote/index.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import s from './styles.module.scss';
+
+export type EditorialQuoteProps = {
+  quote: string;
```
[the remark this answers is not in the document]
Comment (rogervera) [reply packages/brand-components/src/EditorialMolecules/EditorialQuote/index.tsx:5]: I see the case for this, but I'm okay with leaving as is for this PR until a use for custom markup is needed if that's alright with you.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.4.1-alpha.f6cf35.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5eac1398c03268d4065a4ea9--codecademy-styleguide.netlify.app>https://5eac1398c03268d4065a4ea9--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5eac1398c03268d4065a4ea9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.4.2-alpha.9818d2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5eb02658e0d614258c3ed589--codecademy-styleguide.netlify.app>https://5eb02658e0d614258c3ed589--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5eb02658e0d614258c3ed589>Deploy Logs</a></p>

**Pull request merged**